recipe/meta.yaml

```diff
@@ -26,22 +28,33 @@ requirements:
     - python
 
 test:
+  source_files:
+    - gh_src/et_xmlfile/tests
   imports:
     - et_xmlfile
+    - et_xmlfile.incremental_tree
+    - et_xmlfile.xmlfile
   requires:
     - pip
+    - pytest
+    - lxml
   commands:
     - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - python -c "import os,shutil,sysconfig; sp=sysconfig.get_paths().get('purelib') or sysconfig.get_paths().get('platlib'); src=os.path.join('gh_src','et_xmlfile','tests'); dst=os.path.join(sp,'et_xmlfile','tests'); os.makedirs(os.path.dirname(dst), exist_ok=True); shutil.copytree(src, dst, dirs_exist_ok=True)"
+    - PYTHONPATH=$SRC_DIR/gh_src pytest gh_src -l --strict-markers --ignore=gh_src/et_xmlfile/tests/_vendor  # [unix]
+    - set PYTHONPATH=%SRC_DIR%\gh_src && pytest gh_src -l --strict-markers --ignore=gh_src/et_xmlfile/tests/_vendor  # [win]
```

Comment:
This package is pure python. At best running the tests will uncover compatibility issues with newer python version when the import checks are not sufficient. 
I would not run the tests in that case, as it adds complexity to the recipe for little gains.